Packs/ConcentricAI/Integrations/ConcentricAI/ConcentricAI.yml

```diff
@@ -118,7 +121,18 @@ script:
     description: Get's user details
     execution: false
     name: concentricai-get-user-details
-  dockerimage: demisto/python3:3.8.6.12176
+  - arguments:
+    - default: false
+      description: File ID
+      isArray: false
+      name: cid
+      required: true
+      secret: false
+    deprecated: false
+    description: Get's file sharing details
+    execution: false
+    name: concentricai-get-file-sharing-details
```

Comment:
Please add the command outputs.

Reply:
This command results in an array which is represented via Markdown table ! And no outputs are needed for the given command at the moment !